packages/@aws-cdk/aws-kinesisfirehose/package.json

```diff
@@ -73,26 +73,38 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
+    "@aws-cdk/assert-internal": "0.0.0",
+    "@aws-cdk/aws-s3": "0.0.0",
     "@types/jest": "^26.0.24",
     "cdk-build-tools": "0.0.0",
+    "cdk-integ-tools": "0.0.0",
     "cfn2ts": "0.0.0",
-    "pkglint": "0.0.0",
-    "@aws-cdk/assert-internal": "0.0.0"
+    "pkglint": "0.0.0"
   },
   "dependencies": {
+    "@aws-cdk/aws-cloudwatch": "0.0.0",
+    "@aws-cdk/aws-ec2": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
     "@aws-cdk/core": "0.0.0",
+    "@aws-cdk/custom-resources": "0.0.0",
```

Comment:
Do we use this dependency directly somewhere?

Reply:
ugh, not any more, it's for Redshift. will remove